### ISSUE TITLE
Wrong indentation when use tab style

### ISSUE THREAD

**Issue by Keldmar** [pinterest/ktlint#1245]
When using tabs instead of spaces, I constantly get an error in the class constructor, if the parameters are specified vertically, in the example you can see that only the second class is validated, when using spaces there is no such problem

.editorconfig file
```
root = true
[*.{kt,kts}]

indent_style = tab
```

Sample class
```
package somepackage

internal class SomeClass(
				private val someField1: String,
	private val someField2: String,
	private val someField3: String
) {

	val s = 1

	fun someTest(
		someField1: String,
		someField2: String,
		someField3: String
	) {
		/* ignore */
	}
}

internal class SomeClass2(private val someField1: String, private val someField2: String, private val someField3: String) {

	val s = 1

	fun someTest(someField1: String, someField2: String, someField3: String) {
		/* ignore */
	}
}

```
![image](https://user-images.githubusercontent.com/26333514/136213461-90f5725f-508a-4339-8337-d9a4cee3eca5.png)

```
SomeClassTest.kt:4:1: Unexpected indentation (4) (should be 1)
SomeClassTest.kt:4:5: Unexpected indentation (expected 4, actual 4)
SomeClassTest.kt:5:2: Unexpected indentation (expected 4, actual 1)
SomeClassTest.kt:6:2: Unexpected indentation (expected 4, actual 1)
SomeClassTest.kt:12:3: Unexpected indentation (expected 5, actual 2)
SomeClassTest.kt:13:3: Unexpected indentation (expected 5, actual 2)
SomeClassTest.kt:14:3: Unexpected indentation (expected 5, actual 2)
```

**Comment by paul-dingemans** [comment]
There are more known problems related to tab indentations. I have solved most of them a long time ago but it's PR (and several other unpublished PR's which are blocked by it) is till in the process of reviewing.

**Comment by paul-dingemans** [comment]
@Keldmar  @VovaStelmashchuk The PR mentioned before was merged into version 0.43.0 recently. Can you please check whether the tab indentation problems are solved for you when running with that version? Please note, that we have several open problems with this release, but they are not related to tab indentation.

**Comment by paul-dingemans** [comment]
Closing issue as the problem is resolved in a previous release. Please reopen if the issue is not yet solved for you.

**Comment by bfichter** [comment]
I'm still experiencing this issue with 0.43.2

**Comment by paul-dingemans** [comment]
> I'm still experiencing this issue with 0.43.2

@bfichter please add details for your specific problem and setup in a new issue.
